vaadin-login-flow-parent/vaadin-login-flow-integration-tests/src/test/java/com/vaadin/flow/component/login/test/OverlayClassNameIT.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2000-2022 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.vaadin.flow.component.login.test;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import com.vaadin.flow.component.login.testbench.LoginOverlayElement;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.tests.AbstractComponentIT;
+
+@TestPath("vaadin-login/overlay-class-name")
+public class OverlayClassNameIT extends AbstractComponentIT {
+
+    private static final String LOGIN_OVERLAY_WRAPPER_TAG = "vaadin-login-overlay-wrapper";
+
+    @Before
+    public void init() {
+        open();
+    }
+
+    @Test
+    public void openDialog_overlayHasSameClassNames() {
+        findElement(By.id("open")).click();
+
+        waitForElementPresent(By.tagName(LOGIN_OVERLAY_WRAPPER_TAG));
+        LoginOverlayElement overlay = $(LoginOverlayElement.class).first();
+
+        WebElement overlayWrapper = $(LOGIN_OVERLAY_WRAPPER_TAG).first();
```

Comment:
Consider using `overlay.getLoginOverlayWrapper()` from the TestBench API:

https://github.com/vaadin/flow-components/blob/fed476de4cee4b0f885e13e02f268c454cc33013/vaadin-login-flow-parent/vaadin-login-testbench/src/main/java/com/vaadin/flow/component/login/testbench/LoginOverlayElement.java#L39-L41